.devcontainer/base-image/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM mcr.microsoft.com/devcontainers/rust:0.203.7-1
+
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends \
+        cmake \
+    && rm -rf /var/lib/apt/lists/* \
+    && sh -c "$(curl --location https://taskfile.dev/install.sh)" -- -d -b /usr/local/bin
+
+# ========= Install cargo-tools for non-root user (vscode) =========
```

Comment:
I'm guessing this is here for good reason — I added a link to the docs, though I couldn't get to a conclusive answer. 

If it still works without switching users, I would vote to not switch users — unless we have a good reason?

```suggestion
# ========= Install cargo-tools for non-root user (vscode) =========
# Based on https://code.visualstudio.com/docs/devcontainers/create-dev-container#_install-additional-software
```

Reply:
My understanding is that `cargo install` and `task` are usually assumed to be run with user privileges.
So while it may not be necessary to switch from root user here (I may have tried, but I forget), I think it would be more appropriate to do so.

Reply:
You're correct — they use the `vscode` user upstream... https://github.com/microsoft/vscode-dev-containers/blob/5a084a93b0736ea86395ac99019a5b72a00b6341/containers/rust/.devcontainer/base.Dockerfile

Reply:
FYI, that is the old definition.
Current definition is this. (The name `vscode` is retained for compatibility)
https://github.com/devcontainers/images/blob/c087e361571e2200c3b40ed86d14a8ced84c388a/src/rust/.devcontainer/devcontainer.json#L9